### PR TITLE
fix(app): Solve text being cut off on small devices!

### LINKED ISSUE #505: Text is cut off on small devices
### Description

Text is cut off on small devices in AppIntro.

### To Reproduce
Open menu and select help or by the first start of the app AppIntro starts by default.

### Screenshots

grafik
The last line in GERMAN TRANSLATION is cut off.

### Others
- Device: Sony Xperia XA AND Nexus 5 API 30 (from Android Studio Emulator)
- Android Version: 7
- App Version: 0.15.0

### PR DESCRIPTION
**Pull request by robercoding**
### Changes
Added a Custom Fragment with a custom layout to solve text being cut off on small devices. This fragment has the exact same name that had the AppIntro library for its fragment, so you won't notice a change there.

### Others
- Resolves #505

### CODE REVIEW

File: app/src/main/res/layout/fragment_app_intro.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:id="@+id/main"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:paddingTop="@dimen/appintro_statusbar_height"
+  android:paddingBottom="@dimen/appintro2_bottombar_height"
```
Comment (ashutoshgngwr) [inline app/src/main/res/layout/fragment_app_intro.xml:9]: Can we also add some horizontal padding? I noticed one heading fitting itself right to the edges.

Comment (robercoding) [reply app/src/main/res/layout/fragment_app_intro.xml:9]: Sure, I think 20dp of horizontal padding is enough, right?
```
  android:paddingStart="20dp"
  android:paddingEnd="20dp"
```

Comment (ashutoshgngwr) [reply app/src/main/res/layout/fragment_app_intro.xml:9]: yeah, 20 is plenty.

**Review by ashutoshgngwr — APPROVED** [review]
LGTM, thanks! :+1:

**Pull request merged**